framework.go

```diff
@@ -170,17 +171,17 @@ func (f *Framework) ApplyCluster(ctx context.Context, cluster *application.Clust
 	}
 
 	// Apply Cluster resources
-	if err := f.MC().Client.Create(ctx, clusterCM); err != nil {
+	if err := f.MC().CreateOrUpdate(ctx, clusterCM); err != nil {
```

Comment:
Should we just rename it to Apply? Or would you rather be explicit with what the function is doing?

Reply:
I'd rather leave it explicit. It's not strictly speaking the same logic as apply so don't want to mislead.